base/floatfuncs.jl

```diff
@@ -388,6 +385,14 @@ function fma_emulated(a::Float64, b::Float64,c::Float64)
             r = abhi+c
             s = (abs(abhi) > abs(c)) ? (abhi-r+c+ablo) : (c-r+abhi+ablo)
             sumhi = r+s
+            if issubnormal(ldexp(sumhi, bias)) #black magic don't worry about it.
```

Comment:
While perhaps funny, I don't think we should have comments like this :). Someone else reading the code at some later time should have some chance to understand what is going on. Some comments could perhaps help with that.

Reply:
I agree. It would be nice to have an explanatory comment.

Reply:
fair. Fixed.